### PR TITLE
[mathgl] fix deadlock

### PR DESCRIPTION
**Pull request by wtywtykk**
Disables OpenMP parallel in mglFont::FillY12. This function is called in dllmain, so it causes deadlock on windows.

This is not a vcpkg-only issue, so I'm not sure if I should fix here, but this should be quicker than waiting for the upstream update.

- #### What does your PR fix?  
  Fixes deadlock on windows

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  all, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/mathgl/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "mathgl",
   "version-string": "2.4.3",
```
Comment (JonLiu1993) [inline ports/mathgl/vcpkg.json:3]:
```suggestion
  "version": "2.4.3",
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
I think we should give upstream an opportunity to look at this before applying the patch, since as you said it doesn't look like a specific-to-vcpkg problem. If they do not respond after a reasonable amount of time (say 3-4 weeks?) or if they merge this fix themselves then we can land it here. Does that sound reasonable?

(We don't necessarily mind adding patches like this but we tend to want to give upstream at least an opportunity to look since they may have strong opinions about what the fix is and we don't want to speak for them)

**Comment by JonLiu1993** [comment]
@wtywtykk ,Can you please resolve the conflicts against master? maybe we ready to merge this pr

**Comment by wtywtykk** [comment]
> @wtywtykk ,Can you please resolve the conflicts against master? maybe we ready to merge this pr

@JonLiu1993 The upstream author said that he will fix that in v2.5, so I'm checking if v2.5 has fixed this issue. If v2.5 works I think it's better to directly use v2.5.

**Comment by wtywtykk** [comment]
moving to v2.5, this fix is no longer needed

**Pull request closed without merging**